modules/order/src/Element/ProfileSelect.php

```diff
@@ -18,15 +20,14 @@
  *   '#default_value' => $profile,
  *   '#default_country' => 'FR',
  *   '#available_countries' => ['US', 'FR'],
+ *   '#profile_type' => 'customer',
+ *   '#owner_uid' => \Drupal::currentUser()->id(),
  * ];
  * @endcode
- * To access the profile in validation or submission callbacks, use
- * $form['billing_profile']['#profile']. Due to Drupal core limitations the
- * profile can't be accessed via $form_state->getValue('billing_profile').
  *
- * @RenderElement("commerce_profile_select")
+ * @FormElement("commerce_profile_select")
  */
-class ProfileSelect extends RenderElement {
+class ProfileSelect extends FormElement {
```

Comment:
This is reverting the fix from #2857051.

Guessing by accident, since this PR is older than that fix.